CHANGELOG

```diff
@@ -1,3 +1,8 @@
+qtile X.X.X, released XXXX-XX-XX:
+    !!! Config breakage !!!
+	- hooks that pass the qtile object as an argument no longer will. It
```

Comment:
Can you add the list of the hooks here so people can grep?

Reply:
Ah yep good idea